Documentation/Books/Manual/Programs/Arangodump/Maskings.md

```diff
@@ -61,8 +65,78 @@ For example:
     }
 
 In the example the collection "private" is completely ignored. Only the
-structure of the collection "log" is dump, but not the data itself.
-The collection "person" is dumped completed but masking the "name" field when
-it occurs on the to-level. It masks the field "security_id" anywhere in the
-document.
+structure of the collection "log" is dumped, but not the data itself.
+The collection "person" is dumped completely but masking the "name" field if
+it occurs on the top-level. It masks the field "security_id" anywhere in the
+document. See below for a complete description of the parameters of
+"xify_front".
+
+Path
+----
+
+If the path starts with a `.` then it is considered to be a wildcard match.
```

Comment:
```suggestion
If the path starts with a `.` then it will be matched recursively.
```
Maybe better than "wildcard" because that could be understood as `*name` to match every attribute that ends with "name".